p9Layouts/defaultPnorLayout_128.xml

```diff
@@ -365,4 +365,13 @@ Layout Description
         <readOnly/>
         <ecc/>
     </section>
+    <section>
+        <description>Ultravisor binary image (1MB)</description>
+        <eyeCatch>UVISOR</eyeCatch>
+        <physicalOffset>0x2F8E000</physicalOffset>
```

Comment:
Were you intentionally leaving more room for the SBKT partition to grow? Seems like otherwise this should start at 0x2F86000?

Reply:
Ah I see -- In the 64 MB layout we have an HDAT partition that is not in this layout file. @sannerd -- I assume we should add the HDAT partition in this layout as well?

Reply:
moot point, as next commit deletes.